books/free-programming-books-langs.md

```diff
@@ -1389,8 +1389,11 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### Jenkins
 
+* [Jenkins 2: Up and Running](https://theswissbay.ch/pdf/Books/Computer%20science/jenkins2_upandrunning.pdf) (PDF)
```

Comment:
This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines. remove it.